samples/can/README.rst

```diff
@@ -0,0 +1,45 @@
+.. _can-sample:
+
+Controller Area Network
+###########
+
+Overview
+********
+
+This sample demonstrates how to use the CAN API.
+Messages with standard and extended identifiers are sent over the bus triggered
+by a button event.
+Messages are received using message queues and ISRs. Those messages blink LEDs
+or print to the console.
+
+Building and Running
+********************
+
+The reception of messages is indicated by blinking LEDs and messages on the
+console.
+In loopback mode, the board receives its own messages. This could be used for
+standalone testing.
+
+The sample can be built and executed on boards supporting CAN.
```

Comment:
How do we know if the board supports CAN?  Better would be to state the hardware requirements needed on the board.